elastic/kibana
# [Security Solutions] Fix timeline not able to save on investigating alert from dashboard

issue: https://github.com/elastic/kibana/issues/149800

## Summary

The timeline endpoint is returning an error everywhere we call `openTimelineWithFilters` (entity analytics and detections and response pages)

I compare a broken data provider with one that works and spotted the extra `and: []`
<img width="1424" alt="Screenshot 2023-02-20 at 13 54 19" src="https://user-images.githubusercontent.com/1490444/220121799-9d33a0f8-d319-4161-95e2-c9c3fb324972.png">

After removing `and: []` it works.

### How to test it?
* On entity analytics and detections and response pages
* Open the timeline from the alerts column 
* Check if the timeline HTTP call status code is 200
* Save the timeline and check if it is saved


### Checklist

Delete any items that are not applicable to this PR.
